gmso/tests/test_potential.py

```diff
@@ -222,3 +224,13 @@ def test_set_expression_and_params_mismatch(self):
                     'u': 1.0*u.g,
                     'v': 1.0*u.m},
             )
+
```

Comment:
Would it make sense to have a test that instantiates a type of potential (harmonic bond or anything is fine) the traditional way, and then also make one with the template fashion and check if all of their attributes are the same?

Reply:
Ok, I will add the test.